frontend/src/components/role/BindingList.tsx

```diff
@@ -70,6 +70,7 @@ export default function RoleBindingList() {
       <SimpleTable
         rowsPerPage={[15, 25, 50]}
         filterFunction={filterFunc}
+        advancedFilters={['kind']}
```

Comment:
We should add it also to the Workload overview list, as that'll mix a lot of different resource types.

Reply:
done